python/foglamp/common/configuration_manager.py

```diff
@@ -225,9 +233,12 @@ def get_entry_val(k):
 
                 # If Entry item exists in optional list, then update expected item entries
                 if entry_name in optional_item_entries:
-                    if entry_name == 'readonly':
+                    if entry_name == 'readonly' or entry_name == 'deprecated':
                         if self._validate_type_value('boolean', entry_val) is False:
                             raise ValueError('Unrecognized value for item_name {}'.format(entry_name))
+                        if entry_name == 'deprecated':
+                            if entry_val.lower() != 'true':
+                                raise ValueError('Unrecognized value for item_name {}'.format(entry_name))
```

Comment:
even if we RESTRICT  (instead of moving forward  if boolean false) then give clear message? `deprecated item' value can only be true`

Reply:
done